client/my-sites/themes/theme-activation-confirmation-modal.scss

```diff
@@ -0,0 +1,40 @@
+.theme-activation-confirmation-modal {
+	min-height: 90px;
+	max-width: 400px;
+	margin: 0 auto;
+
+	@include breakpoint-deprecated( '<480px' ) {
```

Comment:
Let's replace this with the new Gutenberg/WordPress core breakpoints system: https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#media-queries

Reply:
Done! :)